style.css

```diff
@@ -38,8 +40,8 @@ html, body {
     top: 0;
     left: 0;
     z-index: 50;
-    position: fixed; 
-    background-color: var(--main-bg-color);   
+    position: fixed !important; 
```

Comment:
This line tells me that you do not know what is happening in your own code and you no longer control it. 
This is a huge mistake that will not be allowed in any project, you are no longer able to overwrite this value from anywhere else in code. Please remove important!!!

Reply:
Removed.
I had one problem with header - it made small movements as I scrolled, even though he has position: fixed. I tried to fix this and this line was one of this attempts. I should have deleted it right away.

Reply:
It is not removed.

Reply:
Because it's weekend.
Already pushed it.